Tools/src/Tools/run_hgcroc.cxx

```diff
@@ -0,0 +1,133 @@
+
```

Comment:
Is this suppose to be a test?

Reply:
No. The translation from input voltage to output ADC or TOT is heavily dependent on how the emulator functions and what the input parameters are. For this reason, there isn't a simple test that can be done, but I still think this executable is useful to maintain so that future changes to the emulator can be validated more quickly by isolating the chip emulation from the rest of the digi pipeline.

Reply:
OK makes sense. I'm more wondering if this can be recast as a test?

Reply:
Will say, this is not a blocker but something I'm curious about.

Reply:
We _could_ recast it as a test, but it would only be testing very rudimentary things like the number of hits, so I opted to leave it out of the test framework so that it is easier to run standalone.